tests/end_to_end/bytecodes.py

```diff
@@ -722,7 +716,7 @@
             "value": 0,
             "code": "4300",
             "calldata": "",
-            "stack": str(blockhashes["current_block"]["block_number"]),
+            "stack": "{block_number}",
```

Comment:
Can you just help me understand where the `block_number` comes from? I can't find it anywhere in the code.